features/fields_extensie.feature

```diff
@@ -0,0 +1,46 @@
+# language: nl
+
+Functionaliteit: Aanpasbare representatie met de fields parameter
+  Deze feature is een extensie op https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/fields.feature
+
+  In de Reisdocumenten API wordt de uitgebreide variant van fields ondersteund. Velden in een groep kunnen dus worden opgevraagd zonder het hele pad op te noemen, bijvoorbeeld fields=voornamen levert naam.voornamen.
```

Comment:
de uitgebreide variant van fields is niet relevant voor reisdocumenten. Die is alleen interessant wanneer er veel gegevens in gegevensgroepen zitten. Zoals verblijfplaats bij personen en tenaamstelling bij zakelijkgerechtigden.

Reisdocumenten hebben helemaal geen gegevensgroepen, afgezien van datums en tabellen.

Reply:
Feature-file verwijderd en verwijzing aangepast naar common